packages/orbit-components/src/Textarea/README.md

```diff
@@ -16,32 +16,34 @@ After adding import into your project you can use it simply like:
-| readOnly     | `boolean`                  |              | Adds readOnly to html textarea element.                                                                        |
-| required     | `boolean`                  |              | If true, the label is displayed as required.                                                                   |
-| spaceAfter   | `enum`                     |              | Additional `margin-bottom` after component.                                                                    |
-| tabIndex     | `string \| number`         |              | Specifies the tab order of an element                                                                          |
-| value        | `string`                   |              | Specifies the value of the Textarea. To be used with controlled usage.                                         |
-| helpClosable | `boolean`                  | `true`       | Whether to display help as a closable tooltip, or have it open only while the field is focused, same as error. |
+| Name           | Type                       | Default      | Description                                                                                                    |
+| :------------- | :------------------------- | :----------- | :------------------------------------------------------------------------------------------------------------- |
+| dataAttrs      | `Object`                   |              | Optional prop for passing `data-*` attributes to the `textarea` DOM element.                                   |
+| dataTest       | `string`                   |              | Optional prop for testing purposes.                                                                            |
+| id             | `string`                   |              | Set `id` for `Textarea`                                                                                        |
+| defaultValue   | `string`                   |              | Specifies the default value of the Textarea. To be used with uncontrolled usage.                               |
+| disabled       | `boolean`                  |              | If `true`, the Textarea will be disabled.                                                                      |
+| error          | `React.Node`               |              | The error to display beneath the Textarea. [See Functional specs](#functional-specs)                           |
+| fullHeight     | `boolean`                  | `false`      | If `true`, the Textarea will take 100 % of parent's height.                                                    |
+| help           | `React.Node`               |              | The help to display beneath the Textarea.                                                                      |
+| label          | `Translation`              |              | The label for the Textarea. [See Functional specs](#functional-specs)                                          |
```

Comment:
I would remove `See Functional specs` link, because there's nothing about the `label` in this section now. :D